### ISSUE TITLE
Instructions for puma-dev HTTPS with guard-livereload?

### ISSUE THREAD

**Issue by richardonrails** [puma/puma-dev#326]
Apologies if this is inappropriate to ask here. I'm not quite sure which gem needs enhancing to make this work well...

I'd simply like to use the following together in my Rails 7 app for livereload support in a puma-dev HTTPS app:
- `puma-dev`
- [`guard-livereload`](https://github.com/guard/guard-livereload)
- [`rack-livereload`](https://github.com/jaredmdobson/rack-livereload)

[In this related issue someone described](https://github.com/guard/guard-livereload/issues/143#issuecomment-1341802770) how to make it work when HTTPS isn't coming from `puma-dev` but I'm not sure how to handle this with `puma-dev` HTTPS.

Basically, I'm wondering if `puma-dev` might want to provide instructions in the README for getting it to play well with guard-livereload, similar to the section on Webpack Dev Server. I'm a bit stuck :)
